backend/layers/common/entities.py

```diff
@@ -178,7 +184,7 @@ class CollectionVersion:
     datasets: List[DatasetVersion]
     published_at: Optional[datetime]
```

Comment:
we already maintain published_at in the CollectionVersion, so really we're only pulling in canonical_collection for tombstoned, right? at that point, should we consider just making tombstone as part of CollectionVersion and updating on deletes?

Reply:
correction: _published_at in the canonical collection is the date where the collection was first published_

While not necessary, changing that canonical collection field name to `originally_published_at` would be useful to distinguish

Reply:
@atolopko-czi what naming convention do you prefer?

Reply:
If this timestamp is in fact the first collection version's published_at value, then I like Nayib's suggestion.  These also work `{first,initially}_published_at`, but I'm at a loss to pick a "best" choice. :)

Reply:
`originally_published_at` is the best one imho. I figured the `originally` was implicitly implied by the fact that belongs to the canonical, but as a wise man once said, `explicit is better than implicit`.